requirements.txt

```diff
@@ -0,0 +1 @@
+psycopg2
```

Comment:
It's good practice in Python projects to include any dependent packages in a requirements.txt file with each package on a separate line. Then they can be installed using `pip3 install -r requirements.txt`.